code/chapter-2/visualization.py

```diff
@@ -113,7 +115,6 @@ def join_images(img1, img2):
 
 
 def process_image(image_path, output_path):
```

Comment:
Small suggestion: Can you please update the signature of `process_image` to take in the model as well?

Reply:
I am not very good at python, but the variable ist supposed to be a global variable in the visualization script. Since the `process_image ` function is meant to be called externally I don't think it should be necessary to pass the model.